frontends/mit-open/src/pages/HomePage/BrowseTopicsSection.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react"
+import { Container, styled, theme, Button, Typography } from "ol-components"
+import { Link } from "react-router-dom"
+import { useLearningResourceTopics } from "api/hooks/learningResources"
+import {
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiBriefcaseLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiEarthLine,
+} from "@remixicon/react"
+
+/* TODO Using any icons until we have a solution for specifying them */
+const ICONS = [
+  RiBriefcaseLine,
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiEarthLine,
+]
+
+const Section = styled.section`
+  background:
+    linear-gradient(270deg, rgba(243 244 248 / 0%) 0%, rgb(243 244 248) 100%),
+    url("/static/images/open-bg-texture-wgradient.jpg") lightgray 50% / cover
+      no-repeat;
+  background-position: center right;
+  padding: 80px 0;
+  ${theme.breakpoints.down("md")} {
+    padding: 40px 0;
+  }
+`
+
+const Title = styled(Typography)`
+  text-align: center;
+`
+
+const Topics = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px 24px;
+  margin: 40px 0;
+  ${theme.breakpoints.down("md")} {
+    gap: 5px;
+    margin: 24px 0;
+  }
+`
+
+const TopicBox = styled(Link)`
+  flex: 0 1 calc(100% * (1 / 3) - 16px);
+  padding: 28px 30px;
+  ${theme.breakpoints.down("md")} {
+    flex: 0 1 100%;
+    padding: 18px 15px;
+  }
+
+  border-radius: 5px;
+  border: 1px solid ${theme.custom.colors.lightGray2};
+  background: ${theme.custom.colors.white};
+  overflow: hidden;
```

Comment:
aCan you check with @steven-hatch about a hover state for these links? IMO the background should darken a bit.

Reply:
I've updated the hover state:
<img width="428" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/b6497f32-2af8-460d-8e86-503292066128">